docs/azure_arc_lighthouse/_index.md

```diff
@@ -0,0 +1,132 @@
+Azure Lighthouse allows Contoso to also get monitoring and security alerts across all of the tenant’s subscriptions, run multitenant queries using KQL and set up dashboards that provides valuable insights on the managed environments. There is no need to store logs from different entities into a shared log analytics workspace, Microsoft, Fabrikam and Fourth Coffee can  keep their logs on a dedicated workspace in their subscription, while Contoso gets delegated access to them and get insights from all tenants.
+
+![Azure Monitor multitenant](./06.png)
+
+By enabling a resource in Azure Arc it gives you the ability to perform configuration management and monitoring tasks on those services as if they were first class citizens in Azure. You will be able to monitor your connected machine guest operating system performance or your Kubernetes clusters at the scope of the resource with VM and container Insights.
+
+![Azure Monitor with Azure Arc](./07.png)
+
+Use these Jumpstart scenarios to integrate Azure Monitor and Azure Arc in your managed tenants:
+
+- [Deploy Monitoring Agent Extension on Azure Arc Linux and Windows servers using Extension Management](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/day2/arc_vm_extension_mma_arm/)
+- [Integrate Azure Monitor for Containers with AKS as an Azure Arc Connected Cluster](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_monitor/)
+- [Integrate Azure Monitor for Containers with GKE as an Azure Arc Connected Cluster](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/gke/gke_monitor/)
+- [Integrate Azure Monitor for Containers with MicroK8s as an Azure Arc Connected Cluster](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/microk8s/local_microk8s_monitor/)
+
+### Hybrid Services Onboarding at Scale
+
+In the Contoso scenario, you will centrally manage at scale infrastructure and services that are distributed, for the operations you will usually use services such as zure Monitoring, Azure Automation, Azure Security Center or Azure Sentinel. To work with these services you will need to deploy an agent on your Arc enabled resources and there are several ways you can do that:
+
+- Using a configuration management tool of your choise
+- Using Azure Policy and its remediation tasks
+- Azure Arc’s extension management capabilities
+
+The extension management feature for [Azure Arc enabled servers](https://docs.microsoft.com/en-us/azure/azure-arc/servers/manage-vm-extensions) and [Azure Arc enabled Kubernetes](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/extensions) provides the same post-deployment configuration and automation tasks that you have for Azure VMs or Azure Kubernetes Service. With this feature you can deploy small pieces of software to your Arc enabled resources in an automated fashion, this can be the onboarding on Azure Monitor, Azure Defender, DSC or even the deployment of a custom script on your VMs.
```

Comment:
```suggestion
In the Contoso scenario, you will centrally manage at scale infrastructure and services that are distributed, for the operations you will usually use services such as Azure Monitoring, Azure Automation, Azure Security Center, or Azure Sentinel. To work with these services you will need to deploy an agent on your Arc enabled resources and there are several ways you can do that:

- Using a configuration management tool of your choice
- Using Azure Policy and its remediation tasks
- Azure Arc’s extension management capabilities

The extension management feature for [Azure Arc enabled servers](https://docs.microsoft.com/en-us/azure/azure-arc/servers/manage-vm-extensions) and [Azure Arc enabled Kubernetes](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/extensions) provides the same post-deployment configuration and automation tasks that you have for Azure VMs or Azure Kubernetes Service. With this feature you can deploy small pieces of software to your Arc enabled resources in an automated fashion, this can be the onboarding on Azure Monitor, Azure Defender, DSC, or even the deployment of a custom script on your VMs.
```